dealroom_firestore_connector/batch.py

```diff
@@ -50,6 +51,22 @@ def count_write_wrapper(self, *args, **kwargs):
 
         return count_write_wrapper
 
+    @__count_write
+    def __update_last_edit(self, doc_ref):
+        super().update(doc_ref, {"last_edit": datetime.now()})
+    
+    def __check_if_update_last_edit(self, doc_ref):
+        """If the document reference points to the history collection
+        then update its "last_edit" field to the currrent datetime. This
+        datetame is parsed as a Timestamp in the document.
+
+        Args:
+            doc_ref ([type]): [description]
+        """
+        _path = doc_ref.path.split("/", 1)
+        if len(_path) > 1 and _path[0] == "history":
+            self.__update_last_edit(doc_ref)
+
```

Comment:
could you convert this to single underscore? I know we have already the __count_write but this should also get updated. Thanks!

Reply:
Sure thing, solved in 32cf5ee